README.markdown

```diff
@@ -152,7 +152,7 @@ Upstream bar.com
         127.0.0.1:12356 up
 ```
 
-If an upstream has no health checkers, then it will be marked by `(NO checkers)`, as in
+If you turn on lua_code_cache off or an upstream has no health checkers, then it will be marked by `(NO checkers)`, as in
```

Comment:
Nobody, ever, should use `lua_code_cache off` in production. That will make OpenResty much slower than even Apache and PHP.